llama-index-integrations/readers/llama-index-readers-microsoft-onedrive/llama_index/readers/microsoft_onedrive/base.py

```diff
@@ -653,6 +653,9 @@ def load_data(
                 f"An error occurred while loading the data: {e}", exc_info=True
             )
 
+    def get_permission_info(self, resource_id: str, *args: Any, **kwargs: Any) -> Dict:
+        return self._get_permissions_info(resource_id, *args, **kwargs)
```

Comment:
The types don't really make sense at all here -- resource_id is a string, item is a dict, but you treat them the same -- what is the actual type here? 😅